examples/hbase-snapshot-migration-scripts/restore-snapshot.sh

```diff
@@ -0,0 +1,36 @@
+#! /bin/bash
+# This script restores snapshots on the cluster
+# Run this script on the master node of HBase 2x cluster
```

Comment:
The migration is our intended target, but is this actually specific hbase 2x?

Like this would probably work with any two HBase clusters regardless of version. These scripts might be more generally useful, so we may want to avoid including documentation that limits it.

We may want to keep language general enough with terms like "source" and "target"/"destination" cluster.